docs/source/getting-started/app-architecture.rst

```diff
@@ -53,20 +53,20 @@ In this case all users that a particular user is following are able to see the u
 
 It's also important to distinguish between parties, users, and aliases in terms of naming:
  - Parties are unique across the entire Daml network. These must be allocated before you can use them to log in, and allocation results in a random-looking (but not actually random) string that identifies the party and is used in your Daml code. Parties are a builtin concept.
- - On each participant node you can create users with human-readable user ids. Each user can be associated with a party allocated on that participant node, and refers to that party only on that node. Users are a purely local concept, meaning you can never address a user on another node by user id, and you never work with users in your Daml code; party ids are always used for these purposes. Users are also a builtin concept.
- - Lastly we have user aliases. These are not a builtin concept, they are defined by an Alias template within the specific model used in this guide. Aliases serve as a way to address parties on all nodes via a human readable name.
+ - On each participant node you can create users with human-readable user ids. Each user can be associated with one or more parties (up to 100) allocated on that participant node, and refers to that party only on that node. Users are a purely local concept, meaning you can never address a user on another node by user id, and you never work with users in your Daml code; party ids are always used for these purposes. Users are also a builtin concept.
```

Comment:
The limit is not 100. I think it’s 10k claims but I’m struggling to find the docs.